assembly/src/parsers/nodes.rs

```diff
@@ -285,7 +285,7 @@ pub enum Instruction {
     ExecLocal(u16),
     ExecImported(ProcedureId),
     CallLocal(u16),
-    CallImported(ProcedureId),
```

Comment:
Originally, I was thinking we'd add just another `Call` variant (e.g., something like `CallMastRoot`) - what do you think about it? (i.e., would it complicate or simplify things?)

Reply:
Good catch, I think your proposal is better.